### PR TITLE
Switched out some of the join and leave logging values that didn't make sense.

### PR DESCRIPTION
**Pull request by KiriCattus**
When a user joins it should show when the user joined Discord, while in the same file I replaced the leave embeds "Joined" line with a "Joined Server" and "Joined Discord" line.

### CODE REVIEW

File: src/main/java/net/neoforged/camelot/log/JoinsLogging.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public void onEvent(@NotNull GenericEvent gevent) {
                     .setTitle("User Joined")
                     .addField("User", event.getMember().getUser().getName() + " (" + event.getMember().getAsMention() + ")", true)
                     .addField("Account age", "**" + Duration.ofSeconds(OffsetDateTime.now().toEpochSecond() - event.getUser().getTimeCreated().toEpochSecond()).toDays() + " days**", true)
-                    .addField("Member count", String.valueOf(event.getGuild().getMemberCount()), true)
+                    .addField("Joined Discord", TimeFormat.RELATIVE.format(event.getUser().getTimeCreated()), true)
```
Comment (Matyrobbrt) [inline src/main/java/net/neoforged/camelot/log/JoinsLogging.java:36]: Why did you remove the member count? Also, the account age is when the user joined discord but in days.

**Review by Matyrobbrt — CHANGES_REQUESTED** [review]

File: src/main/java/net/neoforged/camelot/log/JoinsLogging.java [hunk 2]
```diff
@@ -42,7 +42,8 @@ public void onEvent(@NotNull GenericEvent gevent) {
                     .setTitle("User Left")
                     .addField("User", event.getUser().getName(), true)
                     .addField("Account created", TimeFormat.DATE_LONG.format(event.getUser().getTimeCreated()), true)
-                    .addField("Joined", TimeFormat.RELATIVE.format(event.getMember().getTimeJoined()), true)
+                    .addField("Joined Server", TimeFormat.RELATIVE.format(event.getMember().getTimeJoined()), true)
+                    .addField("Joined Discord", TimeFormat.RELATIVE.format(event.getUser().getTimeCreated()), true)
```
[the remark this answers is not in the document]
Comment (KiriCattus) [reply src/main/java/net/neoforged/camelot/log/JoinsLogging.java:46]: getTimeJoined gets the time the user joined the server in question whereas getTimeCreated gets the time the users account was created, see the old /user command we used to have in the Commander bot

[the remark this answers is not in the document]
Comment (Matyrobbrt) [reply src/main/java/net/neoforged/camelot/log/JoinsLogging.java:46]: Check 2 lines above, `Account created` already has the time the user created their Discord account.

[the remark this answers is not in the document]
Comment (KiriCattus) [reply src/main/java/net/neoforged/camelot/log/JoinsLogging.java:46]: ~~Don't rush your work/PR's kids...~~
My aim was to make things consistant and clearly failed xD my apologies
image
